### PR TITLE
Add buggies

### PR DESCRIPTION
**Pull request by kaspar-p**
cars

### CODE REVIEW

**Comment by kaspar-p** [comment]
shut up

**Comment by jason121301** [comment]
Where is the documentation!!

File: src/main/java/Trader.java [hunk 1]
```diff
@@ -98,6 +98,9 @@ public static void main(String[] args) {
                 new Horse(),
                 new Horse(),
                 new Horse(),
+                new VWBug(),
+                new VWBug(),
+                new VWBug("orange", 10000000),
```
Comment (kaspar-p) [inline src/main/java/Trader.java:103]: horrible color choice

File: src/main/java/VWBug.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    @Override
+    public String sound() {
+        return "beep beep";
+    }
+
+    @Override
+    public int getPrice() {
+        return -0;
+    }
```
Comment (kaspar-p) [inline src/main/java/VWBug.java:23]: ????

**Pull request merged**